package.json

```diff
@@ -25,6 +25,7 @@
     "gulp-rename": "2.0.0",
     "gulp-sass": "4.1.0",
     "gulp-size": "3.0.0",
+    "gulp-sourcemaps": "^3.0.0",
```

Comment:
I prefer fixed versions for dev/build things

Note: I know that this is the job of the lockfile and everything, but since some dependencies are old or unmaintained, and some are moving, I prefer set the version, and avoid issues with npm install / update.

Reply:
Ok, this one was added automatically by npm. I will set it to "3.0.0"